project/tychuang1211/README.md

```diff
@@ -0,0 +1,59 @@
+# Improve modmesh camera control 
+
+## Basic Information
+
+Implement a new QT camera controller class to improve current camera control of modmesh.
+
+GitHub repository: https://github.com/tychuang1211/modmesh/tree/70-improve-camera-control
+
+## Problem to Solve
+
+Make camera control more straight forward to use.
+Add functionality:
+1. Resetting.
+2. WSAD.
+3. Change of origin.
+4. ...
+
+## Prospective Users
+
+modmesh users
+
+## System Architecture
+
+Derive a class from `Qt3DExtras::QAbstractCameraController` and implement the virtual function `moveCamera` to suits our needs.
+
+## API Description
+
+> TODO
+
+## Engineering Infrastructure
+
+`Git` for version control
+`Github Action` for running tests
+`Pytest` for testing
```

Comment:
It's a GUI program and your subject is to control the camera.  How do you plan to test it automatically?